File: problemas/iniciante/1132/1132.clj
(def x (read-string (read-line)))
(def y (read-string (read-line)))

(def a (min x y))
(def b (max x y))

(def soma (atom 0))
(doseq [i (range a (+ b 1))]
  (if (== (mod i 13) 0) () (reset! soma (+ @soma i))))

(println @soma)

File: problemas/iniciante/1132/1132.ml
Scanf.scanf "%d\n%d" (fun x y ->
    let a = min x y in
    let b = max x y in
    let soma = ref 0 in
    for i = a to b do
      if i mod 13 != 0 then soma := !soma + i
    done;
    Printf.printf "%d\n" !soma)
